discord-lambda/lambda_function.py

```diff
@@ -0,0 +1,49 @@
+import json
+from nacl.signing import VerifyKey
+from nacl.exceptions import BadSignatureError
+import os
+
+# currently this is VERY VERBOSE we'll need to tone this down for real life
```

Comment:
Out of pure curiosity, where do all these `print()`s go, in the Lambda environment?

Reply:
logs. cloudtrail, specifically. theoretically i should be using a logger, bu this is quicker